dataobj/schedule_entry.h

```diff
@@ -47,7 +47,10 @@ struct schedule_entry_t
 		UNLOAD_ALL        = 1U << 5, // The convoy unloads all loads here.
 		LOAD_BEFORE_DEP   = 1U << 6, // The convoy loads just before the departure.
 		TRANSFER_INTERVAL = 1U << 7,
-	};
+		REVERSE_CONVOY	= 1U << 8,// The reverse order of vehicles in convoy
+		REVERSE_PARENT_CHILDREN		= 1U << 9,// The reverse order of connected convoys
```

Comment:
`REVERSE_CONVOY_COUPLING` may be a better name.
```suggestion
		REVERSE_CONVOY_COUPLING = 1U << 9, // The convoy reverse the parent-child relationship of the convoy coupling.
```

Reply:
Cのenum宣言では全て大文字を使うようにしてください。また、編成は英語で `convoy` です。（convoi表記は既存のconvoi_t型に関連するところ以外では使っていません。）
```c++
REVERSE_CONVOY_COUPLING = 1U << 9,// The convoy reverse the parent-child relationship of the convoy coupling.
```